models/reviewsModels

```diff
@@ -71,3 +71,44 @@ exports.selectComments = (review_id) => {
             });
         });
     }
+
+exports.addComment = (review_id, username, body) => {
+        const idNum = parseInt(review_id);
+        if (Number.isNaN(idNum)) {
+          return Promise.reject({
+            status: 400,
+            msg: 'Bad request'
+          });
+        }
+        
+    return db.query('SELECT * FROM reviews WHERE review_id = $1;', [review_id])
+          .then((review) => {
+            if (review.rows.length === 0) {
+              return Promise.reject({
+                status: 404,
+                msg: 'Please provide a valid review_id'
+              });
+            }
+            
+    return db.query('SELECT * FROM users WHERE username = $1;', [username])
+          .then((user) => {
+                if (user.rows.length === 0) {
+                  return Promise.reject({
+                    status: 404,
+                    msg: 'Invalid Username'
+                  });
+                }
+                
+    return db.query('INSERT INTO comments (review_id, author, body) VALUES ($1, $2, $3) RETURNING *;', [review_id, username, body])
+            .then((result) => {
+                return result.rows;
+                  });
+              });
+          })
+          .catch((err) => {
+            console.log(err);
+            throw err;
+          });
```

Comment:
Im confused as to why there are 3 return db.queries in your model for posting a comment?